dir.traversal.targets

```diff
@@ -1,7 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
-  <Import Project="$(MSBuildToolsPath)\Microsoft.Common.targets" />
-
```

Comment:
This was fixed differently in the corefx repo (see https://github.com/dotnet/corefx/pull/406) should we instead fix it that way or do you think it is better to remove the Microsoft.Common.targets import?